docs/catalyst_cites.bib

```diff
@@ -1,3 +1,44 @@
+@article{CicalaAea2025,
+Author = {Cicala, Steve},
+Title = {Restructuring the Rate Base},
+Journal = {AEA Papers and Proceedings},
+Volume = {115},
+Year = {2025},
+Month = may,
+Pages = {363–68},
+DOI = {10.1257/pandp.20251112},
+URL = {https://www.aeaweb.org/articles?id=10.1257/pandp.20251112}
+}
+
+@article{GowrisankaranJpe2025,
+author = {Gowrisankaran, Gautam and Langer, Ashley and Zhang, Wendan},
+title = {Policy Uncertainty in the Market for Coal Electricity: The Case of Air Toxics Standards},
+journal = {Journal of Political Economy},
+volume = {133},
+number = {6},
+pages = {1757-1795},
+year = {2025},
+doi = {10.1086/734779},
+URL = {https://doi.org/10.1086/734779},
+eprint = {https://doi.org/10.1086/734779},
+abstract = {Government policy uncertainty affects irreversible decisions including technology adoption and exit. This paper quantifies uncertainty surrounding the Mercury and Air Toxics Standards (MATS). We estimate a dynamic oligopoly model for coal-fired electricity generators that recovers generators’ beliefs regarding future MATS enforcement. We develop the approximate belief oligopoly equilibrium concept, where players understand that their decisions impact aggregate market states. MATS enforcement created substantial uncertainty: the perceived enforcement probability dropped to 43\%. Resolving uncertainty early would increase profits by \$1.39 billion but also increase pollution costs by \$0.652–\$1.776 billion. Had exit been unlikely, resolving uncertainty early would have decreased pollution.}
+}
+
+@article{SINGH2025107475,
+title = {Wires and fire: Wildfire investment and network cost differences across {California}’s power providers},
+journal = {The Electricity Journal},
+volume = {38},
+number = {3},
+pages = {107475},
+year = {2025},
+issn = {1040-6190},
+doi = {https://doi.org/10.1016/j.tej.2025.107475},
```

Comment:
I think the DOI field needs to be just the bare DOI, rather than a URL -- this is producing:

[doi:https://doi.org/10.1016/j.tej.2025.107475](https://doi.org/https://doi.org/10.1016/j.tej.2025.107475)

as it tries to construct a DOI URL itself.